### DISCUSSION TITLE
Is there a way to hide the tooltip when hovering over the "+" button on a recipe?

### DISCUSSION THREAD

**Discussion by lamoyrl90** [shedaniel/RoughlyEnoughItems#930] [Q & A]
As the title says, I'd like to know if this is possible. It's covering up some items and I can't tell what I don't have available for some recipes. Not game-breaking, but it makes crafting sometimes a bit more challenging.

**Comment by shedaniel** [comment]
No I don't think so, I think a feature I can have is to aggregate the missing items as a list in the tooltip?
